arcjet-bun/index.ts

```diff
@@ -235,7 +235,17 @@ export default function arcjet<
           ExtraProps<Rules>
         >;
 
-        return aj.protect({}, req);
+        const getBody = async () => {
+          try {
+            const clonedRequest = request.clone();
+            return clonedRequest.text();
+          } catch (e) {
+            log.error("failed to get request body", e);
```

Comment:
This is the wrong way to use the logger. The string needs to have sprintf-style formatting and the error needs to be turned into a string. I'll push the changes.